packages/hint-doctype/src/hint.ts

```diff
@@ -29,8 +29,8 @@ export default class implements IHint {
             line: 0
         };
 
-        const report = async (resource: string, message: string, location = defaultProblemLocation): Promise<void> => {
-            await context.report(resource, message, { location });
+        const report = (resource: string, message: string, location = defaultProblemLocation) => {
```

Comment:
Side effect I wasn't expecting from this change 🎉

This is a pretty big breaking change so we probably want to get #1643 done first before publishing to make sure everything is right.